_config.yml

```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-slate
```

Comment:
💡 We could consider adding these config files to the repo's .gitignore to not check them into git.